README.md

```diff
@@ -73,30 +72,30 @@ as `error` level logs.
 
 ##### Methods
 
-#### `logger.report(level, report[, state])`
+##### `logger.info(log[, state])`
```

Comment:
thinking if we can allow something like:
```logger.info(log[, stateFilter, actionFilter])```
ideally logster should be sending action and state every time by default.